csv/column_scanner_test.go

```diff
@@ -69,6 +69,20 @@ func (this *ColumnScannerFixture) TestColumnNotFound_Panic() {
 	this.So(func() { this.scanner.Column("nope") }, should.Panic)
 }
 
+// TestDuplicateColumnNames confirms that duplicated/repeated
```

Comment:
Slightly out of scope, but as I was documenting the Column method asked myself what would happen if the header had duplicate field names.